src/test/java/dk/kb/image/IIPValidationTest.java

```diff
@@ -0,0 +1,313 @@
+package dk.kb.image;
+
+import dk.kb.util.webservice.exception.InvalidArgumentServiceException;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class IIPValidationTest {
```

Comment:
Overall very thorough checking for problematic values, but you are missing checks for _valid_ values as well as missing (`null`) values.